src/pages/index.tsx

```diff
@@ -189,9 +189,21 @@ export async function getStaticProps(): Promise<StaticProps> {
 
   const filePath = path.join(process.cwd(), 'public', 'json', 'NL.json');
   const fileContents = fs.readFileSync(filePath, 'utf8');
-  const data = JSON.parse(fileContents) as National;
-  const lastGenerated = data.last_generated;
+  const data = JSON.parse(fileContents);
+
+  // Strip away unused data (values) from staticProps
+  // keep last_values because we use them!
+  for (const [key, value] of Object.entries(data)) {
+    if (typeof value === 'object' && value !== null) {
+      for (const [key2] of Object.entries(data[key])) {
+        if (key2 === 'values') {
+          delete data[key].values;
+        }
+      }
+    }
+  }
```

Comment:
I prefer to use named variables that give you a sense of what you're dealing with. Key and value do not tell you much and key2 even less. Maybe something like this :
```suggestion
  for (const [metricName, metric] of Object.entries(data)) {
    if (typeof metric === 'object' && metric !== null) {
      for (const [metricProperty] of Object.entries(data[metricName])) {
        if (metricProperty === 'values') {
          delete data[metricName].values;
        }
      }
    }
  }
```